packages/react-vapor/src/components/limit/Limit.tsx

```diff
@@ -13,6 +13,7 @@ export interface LimitOwnProps {
     id?: string;
     title: string;
     limitLabel?: string;
+    classes?: string;
```

Comment:
Can you name the property `className` instead? I think it would be a better fit with React